### PR TITLE
POL-560 Update SaaS Manager - Suspicious Users - email field and pagination fix

### PR DESCRIPTION
**Pull request by nia-vf1**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt [hunk 1]
```diff
@@ -1,13 +1,13 @@
-name "SaaS Manager - Suspicious Users"
+name "SaaS Manager - Suspicious Users (Nia testing)"
```
Comment (rshade) [inline saas/fsm/suspicious_users/fsm-suspicious_users.pt:1]: Need to fix this

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 rs_pt_ver 20180301
 type "policy"
-short_description "This policy will create an incident when Flexera SaaS Manager identifies suspicious users logging into SaaS applications. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/suspicious_users/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+short_description "This policy will create an incident when Flexera SaaS Manager identifies suspicious users logging into SaaS applications. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/suspicious_users/) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```
Comment (rshade) [inline saas/fsm/suspicious_users/fsm-suspicious_users.pt:4]: Need to update this

**Review by rshade — CHANGES_REQUESTED** [review]

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt (hunk 1 above)
Comment (nia-vf1) [reply saas/fsm/suspicious_users/fsm-suspicious_users.pt]: Done

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt (hunk 2 above)
Comment (nia-vf1) [reply saas/fsm/suspicious_users/fsm-suspicious_users.pt]: Reverted it back to what it said before

**Review by rshade — APPROVED** [review]

**Comment by rshade** [comment]
LGTM

**Pull request merged**